pulsar-io/debezium/oracle/pom.xml

```diff
@@ -31,6 +31,41 @@
   <name>Pulsar IO :: Debezium :: oracle</name>
 
   <dependencies>
+    <dependency>
```

Comment:
we should add those in the <dependencyManagement> section

Reply:
didn't get it, in what section?

Reply:
I think @nicoloboschi means that these should be in dependencyManagement section. I agree. 
Please define these in a dependencyManagement section of `pulsar-io/debezium/pom.xml` file.